dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java

```diff
@@ -1248,7 +1248,8 @@ protected Node methodInvocationConvert(Node node, TypeMirror formalType) {
    * method applies to both method invocations and constructor calls.
    *
    * @param method an ExecutableElement representing a method to be called
-   * @param methodType an ExecutableType representing the type of the method call
+   * @param methodType an ExecutableType representing the type of the method call; viewpoint-adapt
```

Comment:
Is this a description of what is done (it is already viewpoint-adapted) or of what should be done (this method will viewpoint-adapt it)?